features/providers/civil_application_journey.feature

```diff
@@ -294,6 +294,23 @@ Feature: Civil application journeys
     Then I click 'Save and continue'
     Then I should be on a page showing 'Check your answers'
 
+  @javascript @vcr
+  Scenario: I want to change email address from the about financial assessment page
+    Given I complete the journey as far as check your answers
+    Then I click 'Save and continue'
+    Then I am on the benefit check results page
+    Then I click 'Continue'
+    Then I should be on a page showing 'Does your client use online banking?'
+    Then I choose "Yes"
+    Then I click 'Save and continue'
+    Then I am on the About the Financial Assessment page
+    And I click Check Your Answers Change link for 'Email'
+    Then I should be on a page showing 'Email address'
+    Then I fill 'email' with 'test@test.com'
+    Then I click 'Save and continue'
+    Then I am on the application confirmation page
```

Comment:
As my previous comment, I think the next page should be the `About the online financial assessment
` page.
In that case, you could check that the email has been updated:
`Then I am on the About the Financial Assessment page`
`Then I should be on a page showing 'test@test.com'`